testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/health/MicroProfileHealthTestBase.java

```diff
@@ -84,40 +105,44 @@ public void testHealthCheckBeforeDeployment() throws Exception {
         checkGlobalOutcome(managementClient, "check-live", true, null);
 
         // deploy the archive
-        deployer.deploy("MicroProfileHealthTestCase");
+        deployer.deploy(EAR_ARCHIVE_NAME);
     }
 
     @Test
     @InSequence(2)
-    @OperateOnDeployment("MicroProfileHealthTestCase")
+    @OperateOnDeployment(EAR_ARCHIVE_NAME)
     public void testHealthCheckAfterDeployment(@ArquillianResource URL url) throws Exception {
         try (CloseableHttpClient client = HttpClientBuilder.create().build()) {
 
-            checkGlobalOutcome(managementClient, "check", true, "myProbe");
-            checkGlobalOutcome(managementClient, "check-live", true, "myLiveProbe");
-
-            HttpPost request = new HttpPost(url + "microprofile/myApp");
-            List<NameValuePair> nvps = new ArrayList<NameValuePair>();
-            nvps.add(new BasicNameValuePair("up", "false"));
-            request.setEntity(new UrlEncodedFormEntity(nvps));
-
-            CloseableHttpResponse response = client.execute(request);
-            assertEquals(200, response.getStatusLine().getStatusCode());
-
-            checkGlobalOutcome(managementClient, "check", false, "myProbe");
-            checkGlobalOutcome(managementClient, "check-live", false, "myLiveProbe");
+            probeHealth(url, client);
         }
     }
 
     @Test
     @InSequence(3)
-    public void testHealthCheckAfterUndeployment() throws Exception {
+    public void testHealthCheckBetweenDeployments() throws Exception {
```

Comment:
Name of the method is not accurate, `testHealthCheckAfterUndeployment` is still valid